crud/stats/module.lua

```diff
@@ -0,0 +1,211 @@
+local clock = require('clock')
+local checks = require('checks')
+local errors = require('errors')
+local vshard = require('vshard')
+
+local dev_checks = require('crud.common.dev_checks')
+local utils = require('crud.common.utils')
+local op_module = require('crud.stats.operation')
+local registry = require('crud.stats.local_registry')
+
+local StatsError = errors.new_class('StatsError', {capture_stack = false})
+
+local stats = {}
+local is_enabled = false
```

Comment:
Is it resistant to crud hot-reload? Please add a case that stats will be enabled after cartridge hot-reload.

Reply:
Yeah, stats really may be not resistant to crud-reload. I will fix it and then test reload

Reply:
Reworked to support hot reload (both package and cartridge one). The `metrics` has a bug: https://github.com/tarantool/metrics/issues/334 which breaks Cartridge roles reload, so I marked failing tests with xfail and added a workaround so it will only reset collectors observations.